### ISSUE TITLE
Bloat estimation generates many processes when locked

### ISSUE THREAD

**Issue by ced75** [dalibo/temboard#826]
Hi,

I'm using the last TemBoard version on a 12.4 PostgreSQL server.
Many query we're executing are spatial queries with PostGIS and their execution time can be very long, creating temporary tables and modifying many datas.
When executing such long treatments, Temboard generates many connections locked by the PostGIS treatement, and the number of connections is indefinitely increasing as long as the PostGIS queries are not finished.
Here is a screen shot of the activity:
![image](https://user-images.githubusercontent.com/12892581/91053202-c8c74300-e622-11ea-9ae1-0480a5d0138b.png)

A look in pg_locks confirms that the locking query is the PostGIS big query.
Is there a way to avoid Temboard generating locked queries for bloat estimation?

Thank in advance for your help,

Cedric

**Comment by bersace** [comment]
Hmm, that's unfortunate. You may disable `btree_bloat` probe in temboard-agent configuration:

``` ini
[monitoring]
probes = locks,process,replication_lag,wal_files,blocks,replication_connection,db_size,tblspc_size,temp_files_size_delta,sessions,xacts,memory,heap_bloat,loadavg,filesystems_size,cpu,bgwriter                  
```

**Comment by ced75** [comment]
Thanks for your answer.
Is it possible to add the different probes in the documentation?
I've modified the temboard-agent configuration file in the way you indicated and it seems to be OK now.
Here is the TemBoard graphic of sessions illustrating the problem:
![image](https://user-images.githubusercontent.com/12892581/91175352-f07ce080-e6e0-11ea-8f96-8a11d71d79a5.png)
The waiting sessions are increasing as long as the PostGIS query is runing.

**Comment by bersace** [comment]
Yep, we're going to add a timeout statement for this.

**Comment by bersace** [comment]
>  Is it possible to add the different probes in the documentation?

As of now, the best way to know the available probes is to read temboard-agent logs:

```
2020-08-25 12:43:52,709 [ 7570] [probes          ]  INFO: Loaded probe: locks                                                                                                                                        
2020-08-25 12:43:52,709 [ 7570] [probes          ]  INFO: Loaded probe: process                                                                                                                                      
2020-08-25 12:43:52,709 [ 7570] [probes          ]  INFO: Loaded probe: replication_lag                                                                                                                              
2020-08-25 12:43:52,709 [ 7570] [probes          ]  INFO: Loaded probe: wal_files                                                                                                                                    
2020-08-25 12:43:52,709 [ 7570] [probes          ]  INFO: Loaded probe: blocks
2020-08-25 12:43:52,709 [ 7570] [probes          ]  INFO: Loaded probe: replication_connection
2020-08-25 12:43:52,709 [ 7570] [probes          ]  INFO: Loaded probe: db_size
2020-08-25 12:43:52,709 [ 7570] [probes          ]  INFO: Loaded probe: tblspc_size
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: temp_files_size_delta
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: sessions
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: xacts
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: btree_bloat
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: memory
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: heap_bloat
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: loadavg
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: filesystems_size
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: cpu
2020-08-25 12:43:52,710 [ 7570] [probes          ]  INFO: Loaded probe: bgwriter
```

**Comment by ced75** [comment]
Problem solved in Temboard 7.5
Thanks
